webpack.config.js

```diff
@@ -244,6 +244,7 @@ module.exports = (env /*: string */ = 'development', options /*: Options */) =>
   )
 
   if (env === 'production') {
+    // $FlowFixMe
```

Comment:
Not sure what I should have done here instead but figured I could throw a ignore on it and figure it out later.

Reply:
Just curious what the error is?

Reply:
It seems to be something new from [version 0.83](https://github.com/facebook/flow/blob/master/Changelog.md#0820): 

>Removed the ability to use functions as type annotations. This ability was originally designed to support ES3-style classes, but in practice causes confusion and missed errors. We found that the vast majority of added errors were detecting legitimate bugs, and the remainder could be easily updated to ES6 classes.

The error is: 

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ webpack.config.js:247:22

Missing type annotation for T. T is a type parameter declared in array type [1] and was implicitly instantiated at call
of method concat [2].

        244│   )
        245│
        246│   if (env === 'production') {
 [1][2] 247│     config.plugins = config.plugins.concat([
        248│       new BabelMinifyPlugin(),
        249│       new CompressionPlugin({
        250│         test: /\.(js|css)$/
        251│       })
        252│     ])
        253│   }
        254│
        255│   const cssLoader = {
```